front/ampli.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Unfortunately this has to be in the repo, but there is no sensitive information in there.

Reply:
Based on this, it seems doable: https://github.com/aws-amplify/amplify-swift/issues/1662#issuecomment-1059564469

Reply:
@flvndvd the example shown here is about a different project. I think it's completely unrelated to that.
Here is the correct npmjs package: https://www.npmjs.com/package/@amplitude/ampli

This "Ampli.json" object is here to tell the `npx ampli` CLI where we are in terms of properties and event versions types synchronization.